package.json

```diff
@@ -56,12 +56,13 @@
     "react-router-redux": "^4.0.8",
     "react-tooltip": "^3.11.1",
     "react-virtualized": "^9.21.1",
+    "recharts": "^1.8.5",
     "redux": "^3.7.2",
     "redux-thunk": "^2.2.0",
     "sass-loader": "^6.0.6",
     "sqlite3": "^4.1.0",
     "style-loader": "^0.20.1",
-    "web3": "1.0.0-beta.31",
+    "web3": "1.2.4",
```

Comment:
This may have breaking changes for other parts of the app - will need testing and review